staging/src/k8s.io/client-go/tools/events/event_broadcaster.go

```diff
@@ -203,9 +203,9 @@ func recordEvent(sink EventSink, event *v1beta1.Event) (*v1beta1.Event, bool) {
 	var err error
 	isEventSeries := event.Series != nil
 	if isEventSeries {
-		patch, err := createPatchBytesForSeries(event)
-		if err != nil {
-			klog.Errorf("Unable to calculate diff, no merge is possible: %v", err)
+		patch, patchBytesErr := createPatchBytesForSeries(event)
```

Comment:
what about the following?

var patch []byte
patch, err = createPatchBytesForSeries(event)

Reply:
The above seems to be equivalent to what I have.
Using secondary error variable is commonly used practice in such scenario.